2017/index.dd

```diff
@@ -4,6 +4,8 @@ $(T h1,,DConf 2017$(BR) Bigger, Badder, and Berliner!)
 
 $(T h2,,Please join us at DConf 2017, the international conference of the D programming language in Berlin, Germany, May 4-7 2017.)
 
+$(P DConf 2017 is now underway! If you aren't with us in Berlin, be sure to join us via the livestream at $(HTTP youtube.com/watch?v=MqrJZg6PgnM&utm_content=buffercc4c1&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer, YouTube)).
```

Comment:
I don't think there needs to be anything in the URL after the first ampersand (though it doesn't break anything)